liota/lib/utilities/filters/windowed_filters/windowed_range_filter.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
```

Comment:
I believe in spite of adding this as the separate filter it should happen in the example code. In the UDM both filters(Window as well as value filter) should be applied, which will be more elaborative.